src/analyze/BuildSystem.re

```diff
@@ -302,6 +308,10 @@ let getCompiler = (rootPath, buildSystem) => {
     | Dune(Opam(switchPrefix)) =>
       let%try_wrap binPath = getOpamLibOrBinPath(rootPath, switchPrefix, "bin" /+ "ocamlopt.opt");
       binPath
+    | Merlin(rootPath) =>
+      // Assume OCaml is installed next to the project we are building
+      // This is due to BuckleScript shipping an OCaml compiler next to jscomp/
+      RResult.Ok(rootPath /+ ".." /+ "ocaml" /+ "ocamlc.opt")
```

Comment:
Is there any way to not hardcode this? Can it be determined from the `.merlin` file, ideally?

Reply:
Can you explain any further? I don't really know anything about Merlin

Reply:
@bobzhang @jaredly @cristianoc do any of you understand what I'd need to do for this? I need some guidance.

Reply:
Can the path of ocaml compiler be configured so that it does not have to be hard coded here?